softlearning/environments/adapters/dm_control_adapter.py

```diff
@@ -0,0 +1,144 @@
+"""Implements an adapter for DeepMind Control Suite environments."""
+
+from collections import OrderedDict
+
+import numpy as np
+from dm_control import suite
+from dm_control.rl.specs import ArraySpec, BoundedArraySpec
+from gym import spaces
+
+from .softlearning_env import SoftlearningEnv
+
+
+# TODO(hartikainen): Add information of the available environments.
+# See `gym_adapter.py` for example.
+DM_CONTROL_ENVIRONMENTS = {}
+
+
+def convert_dm_control_to_gym_space(dm_control_space):
+    if isinstance(dm_control_space, BoundedArraySpec):
+        gym_box = spaces.Box(
+            low=dm_control_space.minimum,
+            high=dm_control_space.maximum,
+            shape=None,
+            dtype=dm_control_space.dtype)
+        assert gym_box.shape == dm_control_space.shape, (
```

Comment:
Add a comment here about why we do this.
```
# Note: `gym.Box` doesn't allow both shape and min/max to be defined at the same time. Thus we omit shape in the constructor and verify that it's been implicitly set correctly.
```

Reply:
Added. Will commit soon.